src/terminal.py

```diff
@@ -74,7 +78,10 @@ def main():
         sub_names, sub_urls = sub_list_folders(slides_path)
         sub_links = get_all_links(sub_urls, session)
         sorted_links = sorting_links(sub_links)
-        updates.main(session, sub_names, sorted_links, slides_path)
+        upMain = updates.main(session, sub_names, sorted_links, slides_path)
+        display.dashify(upMain)
```

Comment:
Its a good practice to use function names that make their use case apparent.
Use a better name than 'dashify'